src/main/java/frc/robot/RobotContainer.java

```diff
@@ -90,4 +123,9 @@ public void loadCommandSchedulerShuffleboard() {
     CommandScheduler.getInstance().onCommandInterrupt(command -> Shuffleboard.addEventMarker("Command interrupted", command.getName(), EventImportance.kNormal));
     CommandScheduler.getInstance().onCommandFinish(command -> Shuffleboard.addEventMarker("Command finished", command.getName(), EventImportance.kNormal));
   }
+
+  public void shuffleboardUpdates(){
+    m_drive.updateDriveModuleFeedforwardShuffleboard();
+    m_drive.updateDriveModuleFeedforwardShuffleboard();
```

Comment:
I think this can happen in the drivetrain's periodic(), if it is necessary. I also think there is a better way to do it than to run this periodically, like something that come to mind is putting an instant command array on shuffleboard that sets the new module. That way it is just called when the thing is changed, by clicking the button, rather than having to check every 20ms if the value changed.

Reply:
moved to perodic for now. was thinking we could call shuffleboard modules automaticly per test. also instant commands will coss problems if someone calls it twice on acident.